pyproject.toml

```diff
@@ -206,7 +206,10 @@ docstring-code-format = true
 # YTT: flake8-2020
 exclude = ["**/tests/**"]
 explicit-preview-rules = true
-extend-select = ["F822"]  # TODO(egparedes): remove when not longer in preview
+extend-select = [
+  'F822',  # TODO(egparedes): remove when not longer in preview
+  'PLW0133'  # [useless-exception-statement]
+]
```

Comment:
Is there a convention for what goes in `extend-select` and what goes in `select`? I think I understand the difference between the two, but I don't understand e.g. why F822 is in `extend-select` and why FA100 is in `select`...

Reply:
The reason why `F822` goes in `extend-select` is a bit hacky and arbitrary. This rule comes from an active ruleset (`F`) but  still needs to be explicitly activated because it is a rule in `preview` mode and `explicit-preview-rules=true` (to avoid loading all preview rules by default). Since the explicit activation of this rule will become unnecessary as soon as it is removed from preview mode, adding it to `extend-select` was a hack to try to make more visible that it was a temporary activation.

In #839 , this rule activation has been moved to `select` as expected.

Reply:
I would avoid keep this PR as a simple fix for the missing `raise` statements, and activate more ruff rulesets in other PRs like #839